aws-cloudwatch-logging/build.gradle.kts

```diff
@@ -4,9 +4,14 @@ plugins {
 
 dependencies {
     api(projects.micronautAwsSdkV2)
-    implementation(libs.logback.json.classic)
     api(libs.awssdk.cloudwatchlogs)
     api(mn.micronaut.runtime)
     api(mnSerde.micronaut.serde.jackson)
+
+    implementation(mnLogging.logback.classic)
+    implementation(libs.logback.json.classic) {
+        exclude(group = "ch.qos.logback")
```

Comment:
I think you will need explicit excludes for each module for this to work for Maven

Reply:
Do you have an example? 🤔

Reply:
not sure what you mean, you need to specify both the group and the module id